### PR TITLE
Document options for generating non-digest assets

### PR DESCRIPTION
**Pull request by mvz**
This documents the options suggested in https://github.com/rails/sprockets-rails/issues/343#issuecomment-215241277 and https://github.com/rails/sprockets-rails/issues/343#issuecomment-215234763 so people can hopefully find them more quickly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +150,13 @@ The following plugins provide some extras for the Sprockets Asset Pipeline.
 * `config.assets.manifest` (if used) must now include the manifest filename, e.g. `Rails.root.join('config/manifest.json')`. It cannot be a directory.
 * Two cleanup tasks: `rake assets:clean` is now a safe cleanup that only removes older assets that are no longer used, while `rake assets:clobber` nukes the entire `public/assets` directory. The clean task allows for rolling deploys that may still be linking to an old asset while the new assets are being built.
 
+### But what if I want sprockets to generate non-digest assets?
+
+You have several options:
+
+* Use the [non-stupid-digest-assets gem](https://github.com/alexspeller/non-stupid-digest-assets).
```
Comment (rafaelfranca) [inline README.md]: Sorry, but I really don't want to recommend a gem with that name. Can you remove it?

Comment (mvz) [reply README.md]: I suppose I could fork it and give it a better name. It doesn't look that well-maintained either.

Do you still think a gem with that functionality is a reasonable solution?

Comment (mvz) [reply README.md]: I'm currently researching some other options and will update this PR accordingly.

Comment (mvz) [reply README.md]: Updated.

Comment (rafaelfranca) [reply README.md]: yes, I do think that gem is a reasonable solution

**Comment by rafaelfranca** [comment]
Thanks!

**Comment by mvz** [comment]
Thanks for merging this @rafaelfranca.

**Pull request merged**